README.md

```diff
@@ -145,7 +154,7 @@ See [DOCUMENTATION.md](DOCUMENTATION.md).
 
 ## What's missing
 
-Currently, the store does not handle `links` and `meta` attributes.
+Currently, the store does not handle `links` attributes.
```

Comment:
So here, we should say that it handles the top level meta property, but that it does not handle resource-level or relationship-level meta.